### PR TITLE
Add GoF and AW to branch info

### PR DESCRIPTION
**Pull request by acha-bill**
# Description of change

Add GoF and AW to branch info

Screenshot 2021-12-03 at 12 22 40

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

#1803

### CODE REVIEW

File: plugins/dagsvisualizer/visualizer.go [hunk 1]
```diff
@@ -159,9 +160,20 @@ func registerBranchEvents() {
 		})
 	})
 
+	branchWeightChangedClosure := events.NewClosure(func(e *tangle.BranchWeightChangedEvent) {
+		visualizerWorkerPool.TrySubmit(&wsMessage{
+			Type: MsgTypeBranchWeightChanged,
+			Data: &branchWeightChanged{
+				ID:     e.BranchID.Base58(),
+				Weight: e.Weight,
```
Comment (jkrvivian) [inline plugins/dagsvisualizer/visualizer.go:168]: When weight changes, GoF is changed, too.
Let's grab the new GoF and include it here.

File: plugins/dagsvisualizer/frontend/src/index.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import UTXOStore from 'stores/UTXOStore';
 import BranchStore from 'stores/BranchStore';
 import GlobalStore from 'stores/GlobalStore';
 import {Root} from 'components/Root';
+import React from 'react';
```
Comment (jkrvivian) [inline plugins/dagsvisualizer/frontend/src/index.tsx:8]: This package is not used.

File: plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx [hunk 3]
```diff
@@ -1,16 +1,18 @@
 import { action, makeObservable, observable, ObservableMap } from 'mobx';
-import { registerHandler, unregisterHandler, WSMsgType } from 'WS';
+import { registerHandler, unregisterHandler, WSMsgType } from '../WS';
```
Comment (jkrvivian) [inline plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx:2]: The baseUrl is set to `src` in tsconfig.json, `WS` should work. :thinking:

File: plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx [hunk 4]
```diff
@@ -85,7 +93,7 @@ export class BranchStore {
 
         if (this.paused) {
             this.branchToAddAfterResume.push(branch.ID);
-        } 
+        }
```
Comment (jkrvivian) [inline plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx:96]: Thanks for cleaning up extra spaces. :heart:

File: plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx (hunk 3 above)
Comment (acha-bill) [reply plugins/dagsvisualizer/frontend/src/stores/BranchStore.tsx:2]: For some reason `ws` was conflicting with some node_modules stuff. I think we should leave it as `../ws` to be safe.

File: plugins/dagsvisualizer/frontend/src/index.tsx (hunk 2 above)
Comment (acha-bill) [reply plugins/dagsvisualizer/frontend/src/index.tsx:8]: It's to fix TS error: `TS2686: 'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.`. 
Apparently some packages needs it.

**Review by jkrvivian — APPROVED** [review]

**Pull request merged**